joern-cli/frontends/x2cpg/src/main/scala/io/joern/x2cpg/utils/LinkingUtil.scala

```diff
@@ -41,11 +34,16 @@ trait LinkingUtil {
   def nodesWithFullName(cpg: Cpg, x: String): mutable.Seq[NodeRef[_ <: NodeDb]] =
     cpg.graph.indexManager.lookup(PropertyNames.FULL_NAME, x).asScala
 
+  protected def getBatchSize(totalItems: Int): Int =
+    Math.max(totalItems / ConcurrentTaskUtil.MAX_POOL_SIZE, MAX_BATCH_SIZE)
```

Comment:
Why `max`? This would choose `totalItems / ConcurrentTaskUtil.MAX_POOL_SIZE` instead of `MAX_BATCH_SIZE` if `totalItems / ConcurrentTaskUtil.MAX_POOL_SIZE > MAX_BATCH_SIZE`

Reply:
Typing mistake